addOns/openapi/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Changed
+- Fix null pointer exception for OpenAPI import with empty dictionary (Issue 7808)
```

Comment:
- This should be under `### Fixed`.
- End the sentence with a dot.
- `for OpenAPI import` → `when importing a definition`, this add-on applies only to OpenAPI.

Reply:
The dot could still be added.

Reply:
Moved under ### Fixed and updated message + ended with a dot.